eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Snap.hs

```diff
@@ -109,7 +110,7 @@ snapTransition = do
   pure $
     SnapShots
       { ssStakeMark = istakeSnap
-      , ssStakeMarkPoolDistr = calculatePoolDistr istakeSnap
+      , ssStakeMarkPoolDistr = PoolDistr mempty -- calculatePoolDistr istakeSnap
```

Comment:
guessing you didn't mean to leave this here

Reply:
Yeah, I'll revert this one. It was an experiment that I did to make sure it is not causing a slow down in #3394

Reply:
Interestingly enough this change of setting `ssStakeMarkPoolDistr` had a positive affect on performance. I can't really explain why, considering it is not used anywhere. 

I will revert it regardless, because this is necessary for #3390